src/test/java/com/github/awwkoala/selenium/CheckBoxesTests.java

```diff
@@ -0,0 +1,36 @@
+package com.github.awwkoala.selenium;
+
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class CheckBoxesTests {
+/*  Stwórz test automatyczny na podstawie poniższego scenariusza testowego:
+  Przejdź na stronę checkboxes
+  Sprawdź, że po wejściu na stronę checkbox 1 jest odznaczony, zaś checkbox 2 jest zaznaczony
+  Zaznacz checkbox 1, odznacz checkbox 2
+  Sprawdź, że checkbox 1 jest zaznaczony, zaś checkbox 2 jest odznaczony*/
+
+  @Test
+  public void checkBoxesTest() {
+    System.setProperty("webdriver.chrome.driver", "/opt/WebDriver/bin/chromedriver");
+    WebDriver driver = new ChromeDriver();
+    driver.get("http://theinternet.przyklady.javastart.pl/checkboxes");
```

Comment:
Most of testers while using a selenium is using Page Object Pattern. So in Short you could create `CheckboxesPage` or sth like that. Let that Page class have webDriver property injected via constructor. All driver calls should be wrapped into Page class.

Reply:
`Sprawdź, że po wejściu na stronę checkbox`

Reply:
For me it's //given section